moveit_ros/perception/CMakeLists.txt

```diff
@@ -1,91 +1,81 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(moveit_ros_perception)
 
-if(NOT "${CMAKE_CXX_STANDARD}")
-  set(CMAKE_CXX_STANDARD 14)
-endif()
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
-
-option(WITH_OPENGL "Build the parts that depend on OpenGL" ON)
+# Common cmake code applied to all moveit packages
+find_package(moveit_common REQUIRED)
+moveit_package()
 
-if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE Release)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wpedantic)
 endif()
 
 find_package(Boost REQUIRED thread)
 
-if(WITH_OPENGL)
-  # Prefer newer vendor-specific OpenGL library
-  if(POLICY CMP0072)
-    cmake_policy(SET CMP0072 NEW)
-  endif()
-  find_package(OpenGL REQUIRED)
-  find_package(GLEW REQUIRED)
-  find_package(GLUT REQUIRED)
-  set(gl_LIBS ${gl_LIBS} ${OPENGL_LIBRARIES})
-  set(perception_GL_INCLUDE_DIRS "mesh_filter/include" "depth_image_octomap_updater/include")
-  set(SYSTEM_GL_INCLUDE_DIRS ${GLEW_INCLUDE_DIR} ${GLUT_INCLUDE_DIR})
-endif()
+# if(WITH_OPENGL)
```

Comment:
Please file an issue to enable OpenGL/depth_image_octomap_updater

Reply:
An issue #327 has been filed to track the porting progress of moveit_ros_percpetion.